src/preprocessing.py

```diff
@@ -86,6 +86,43 @@ def preprocessing(df, country_metric_df):
     df = df.drop_duplicates()
     return df
 
+def create_security_test_features(api_df):
+    """ Create new features corresponding to security tests. The following features are being created:
+    - security_test_conducted (0: No, 1: Yes)
+    - 6 columns corresponding to each security test (0: PASS, 1: FAIL, 0.5: NOT AVAILABLE)
+    Parameters
+    ----------
+    api_df: Pandas Dataframe
+        Input dataframe with raw data columns, including security_test_results
+    Returns
+    -------
+    feat_security_tests_df : Pandas Dataframe
+        dataframe with endpoint_id and new security test features
+    """
+    # Data extraction and cleaning
+    security_test_df = api_df[["api_endpoint_id", "security_test_category", "security_test_result (FALSE=Passed; TRUE=Failed)"]]
```

Comment:
this line throw an error, couldn't find the column name

Reply:
I know the error. It's because we change the name of the "security_test_result (FALSE=Passed; TRUE=Failed)" in the earlier lines 34 to 37